funnel/templates/notifications/comment_report_received_email.html.jinja2

```diff
@@ -1,14 +1,26 @@
-{%- extends "notifications/layout_email.html.jinja2" %}
+{%- extends "notifications/layout_email.html.jinja2" -%}
+{%- from "notifications/macros_email.html.jinja2" import hero_image, cta_button -%}
 {%- block content -%}
 
-  <p>
-    {%- trans %}A comment has been reported as spam{% endtrans -%}
-  </p>
+    {# Hero image centered : BEGIN #}
+    {{ hero_image("https://images.hasgeek.com/embed/file/c914f2324f1e4b449beb15b272ce9f10?size=196x149") }}
+    {# Hero image centered : END #}
 
-  <p class="btn-bar">
-    <a class="btn" href="{{ url_for('siteadmin_review_comment', report=view.report.uuid_b58) }}">
-      {%- trans %}Review comment{% endtrans -%}
-    </a>
-  </p>
+    {# Email body : BEGIN #}
+    <tr align="center">
+        <td class="text-container">
+            <h1>Spam alert!</h1>
+            <p>
+                {%- trans %}A comment has been reported as spam{% endtrans -%}
+            </p>
+        </td>
+    </tr>
+    <br>
+
+    {# Button : BEGIN #}
+    {{ cta_button(url_for('siteadmin_review_comment', report=view.report.uuid_b58), "Review comment" )}}
```

Comment:
`gettext` here.

Reply:
Changed across all instances where a string is passed to a macro.